src/test/resources/settings/git-changelog-test-settings.json

```diff
@@ -1,7 +1,7 @@
 {
  "fromRepo": ".",
- "fromRevision": "0000000000000000000000000000000000000000",
```

Comment:
`fromRevision` and `toRevision` should be kept as is here right?

Reply:
Yes. I just fixed, I guess.